config/routes.rb

```diff
@@ -30,7 +30,10 @@
         get "record-template", on: :collection
       end
 
-      get "/consent/confirm", to: "consent_responses#confirm", on: :member
+      resource :consent_responses, path: "consent" do
+        post "confirm"
```

Comment:
I'm guessing `/confirm` will be a GET eventually, and it's a POST for now only because it's actually modifying data, is that right?

Reply:
Yes